src/app/teams/teams.utils.ts

```diff
@@ -1 +1,10 @@
 export const memberCompare = (member1, member2) => member1.userId === member2.userId && member1.userPlanetCode === member2.userPlanetCode;
+
+export function compareFunc(member1, member2) {
+  const name1 = member1.userDoc.doc.lastName;
+  const name2 = member2.userDoc.doc.lastName;
```

Comment:
There are situations where `userDoc` doesn't exist, and this will throw a breaking error.  You should handle that case and return `member2.userId` instead.

Reply:
`(member1.userDoc && member1.userDoc.doc.lastName)`